InternxtDesktop/AppDelegate.swift

```diff
@@ -450,6 +450,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             display()
         }
         self.scheduledManager.resumeBackupScheduler()
+        Task { await usageManager.updateUsage() }
```

Comment:
Let's debounce this, @patricioxavier8 , so a user cannot rageclick on the widget and trigger multiple usage requests, that would be huge for the server

Max 1 usage update every 15 seconds